diff --git a/android/build.gradle b/android/build.gradle
index c8f7fd443cefa148ce2888bea1ef9a5725c82969..e818323290a694fa0e53404a6592d537391ae371 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -18,6 +18,8 @@ def safeExtGet(prop, fallback) {
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
+    namespace "com.lugg.RNCConfig"
+
     defaultConfig {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
